Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'sass-rails', '>= 6'
 gem 'webpacker', github: 'rails/webpacker', ref: 'b6c2180'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'i18n-tasks', '~> 0.9.34'
```

Comment:
I'm 90% sure we should move this under `group :development, :test` below since it shouldn't be needed when running in prod.

Reply:
done in https://github.com/JohnRDOrazio/marriage-booklet/commit/5937e6003b40e4a6fd772335e9ce270d598a3f95